pkg/services/serviceaccounts/database/token_store.go

```diff
@@ -30,6 +33,10 @@ func (s *ServiceAccountsStoreImpl) ListTokens(
 			sess = sess.Where("api_key.service_account_id=?", *query.ServiceAccountID)
 		}
 
+		if query.TokenExpirationDate != nil {
+			sess = sess.Where("api_key.expires - CAST(strftime('%s', api_key.created) AS INT) > ? * 24  * 60 * 60", *query.TokenExpirationDate)
```

Comment:
Does `strftime` work with MySQL, Postgres as well?

Reply:
it's a good point. We do test the different database setups in our test suites, however the current tests do not cover a token expiration date.

First step could be to add a test for this in the pkg/services/serviceaccounts/database/token_store_test.go

wdyt?